zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -635,6 +641,12 @@ impl<F: Field> ExecutionGadget<F> for CallOpGadget<F> {
             callee_code_hash,
             Word::random_linear_combine(*EMPTY_HASH_LE, block.randomness),
         )?;
+        self.is_precompile.assign(
+            region,
+            offset,
+            F::from_u128(code_address.as_u128()),
```

Comment:
will panic here.

try: 
            Word::random_linear_combine(code_address.to_le_bytes(), block.randomness),   ??

Reply:
inside circuit, you compare rlc(code_address) vs 9, so you should assign rlc(code_address) and 9 to the is_precompile(LT) gadget.

Reply:
wait should not it be code_address < 0x0a or code_address <= 0x9 instead of code_address < 0x9 ?

Reply:
this gadget only checks code_address in [0, 9] but in fact you need [1, 9].  Be careful here. Check the code to make sure there is no soundness problem.

Reply:
or you can check rlc(code_address) - 1 in [0, 9)